spec/rubocop/cop/rspec/return_from_stub_spec.rb

```diff
@@ -176,15 +176,15 @@
       RUBY
     end
 
-    it 'ignores stubs without return value' do
+    it 'does not register an offense for stub without return value' do
```

Comment:
Nitpick.
'does not register an offense for A stub without return value' or
'does not register an offense for stubs without return values'.